pkgs/development/python-modules/hdbscan-with-cosine-distance/default.nix

```diff
@@ -0,0 +1,32 @@
+
+{ lib
+, buildPythonPackage
+, cython
+, numpy
+, scipy
+, scikitlearn
+, pythonOlder
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "hdbscan-with-cosine-distance";
+  version = "0.8.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0k6wk8kp2rqm3zsnkndfzmlavias5ij6k1nlk8fk4kl792wzv7bj";
+  };
+
+  # No tests in archive
+  doCheck = false;
+   
+  propagatedBuildInputs = [ cython numpy scipy scikitlearn ];
+
+  meta = with lib; {
+    description = "HDBSCAN - Hierarchical Density-Based Spatial Clustering of Applications with Noise. HDBSCAN is a clustering algorithm with a scikit-learn compatible API; it doesn't require defining the number of clusters sought, and fares well with oddly shaped clusters.";
```

Comment:
This is to long, might be used as `longDescription`. Please have a look at https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes.